### PR TITLE
Update dependency resolvelib to latest

### PR DESCRIPTION
**Pull request by bennati**
It includes fix for backjumping, see
https://github.com/sarugaku/resolvelib/pull/113

Relates-to: #106  

Signed-off-by: Bennati, Stefano <stefano.bennati@here.com>

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -64,7 +64,7 @@ install_requires =
     pkginfo2 >= 30.0.0
     pip-requirements-parser >= 32.0.1
     requests >= 2.18.0
-    resolvelib >= 0.8.1
+    resolvelib @ git+https://github.com/sarugaku/resolvelib/@main
```
Comment (pombredanne) [inline setup.cfg:67]: This is quite a leap of faith! could you instead pin it to a commit?

Comment (bennati) [reply setup.cfg]: done

Comment (bennati) [reply setup.cfg]: updated now to version 1.0.0, which includes the fix

**Comment by bennati** [comment]
can you please create a new release that includes this change, so that we can update scancode dependencies as well?

**Comment by bennati** [comment]
Thanks, can you please create a new release with this change?

**Comment by TG1999** [comment]
@bennati released it here https://pypi.org/project/python-inspector/0.9.6/ thanks for your contribution.

**Pull request merged**